test/docx/body-reader.tests.js

```diff
@@ -567,6 +567,42 @@ test("w:tblHeader marks table row as header", function() {
     }));
 });
 
+test("w:firstRow marks table row as header", function() {
+    var tableXml = new XmlElement("w:tbl", {}, [
+        new XmlElement("w:tr", {}, [
+            new XmlElement("w:trPr", {}, [
+                new XmlElement("w:firstRow")
+            ])
+        ]),
+        new XmlElement("w:tr")
+    ]);
+    var result = readXmlElementValue(tableXml);
+    assertThat(result, isTable({
+        children: contains(
+            isRow({isHeader: true}),
+            isRow({isHeader: false})
+        )
+    }));
+});
+
+test("w:firstColumn marks table column as header", function() {
+    var tableXml = new XmlElement("w:tbl", {}, [
+        new XmlElement("w:tr", {}, [
+            new XmlElement("w:trPr", {}, [
+                new XmlElement("w:firstColumn")
```

Comment:
It is getting applied to the cell isn't it?

`w:tr` is the row
`w:trPr` is the row properties (I think)
`w:firstColumn` is the cell

I couldn't get the tests working though so this was just a best guess based on your already existing code.